provider/src/main/kotlin/org/gradle/kotlin/dsl/NamedDomainObjectContainerExtensions.kt

```diff
@@ -79,24 +95,56 @@ class NamedDomainObjectContainerScope<T : Any>(
     fun <U : T> creating(type: KClass<U>) =
         polymorphicDomainObjectContainer().creating(type)
 
+    /**
+     * Provides a property delegate that creates elements of the given [U].
+     */
+    inline fun <reified U : T> creating() =
```

Comment:
:x: `inline` modifier goes in the line above.

Reply:
Dang, I accidentally ran the ktlint formatter I think, oops.